config/production.template.json

```diff
@@ -4,8 +4,6 @@
   "mongodb": "mongodb://mongodb:27017/youpin",
   "public": "../public/",
   "auth": {
-    "successRedirect": "your url for success login",
-    "failureRedirect": "your url for fail login",
```

Comment:
That's what I thought too but I didn't see anyone setting it yet on production...
So, I thought they were not needed.

Anyway, added them back. PTAL.